open-api/rest-catalog-open-api.yaml

```diff
@@ -1630,6 +1758,76 @@ components:
         - $ref: '#/components/schemas/OAuthClientCredentialsRequest'
         - $ref: '#/components/schemas/OAuthTokenExchangeRequest'
 
+    ScanReportRequest:
```

Comment:
I think we should make this generic and use `anyOf` to wrap `ScanReport` for a generic `metrics` endpoint. If we do that, then we'll need a `report-type` field or something similar in each report object.

Reply:
makes sense, updated the code